lib/ansible/modules/cloud/vmware/vmware_host_sriov.py

```diff
@@ -184,117 +184,112 @@ def __init__(self, module):
             self.module.fail_json(msg="Failed to find host system.")
         self.results = {"before": {}, "after": {}, "changes": {}}
 
-    def sanitize_params(self, before, hostname, vmnic):
+    def sanitize_params(self):
+        """checks user input, raise error if input incompatible
+        :return : None
         """
-        :before     : dict, of params on target interface before changing
-        :hostname   : str, hosthame
-        :vmnic      : srt, target nic
-        :return     : dict, of params forwarded to vCenter, or raise fail in case iconsistancy
-        """
-        params_to_change = {
-            "sriovEnabled": None,
-            "numVirtualFunction": None,
-            "msg": "",
-            "change": False,
-            "changes": {},
-        }
-        # change = False
+
         if self.num_virt_func < 0:
             self.module.fail_json(msg="allowed value for num_virt_func >= 0")
-
         if self.num_virt_func == 0:
             if self.sriov_on is True:
                 self.module.fail_json(
                     msg="with sriov_on == true,  alowed value for num_virt_func > 0"
                 )
-            params_to_change.update({"sriovEnabled": False, "numVirtualFunction": 0})
+            self.sriov_on = False  # fill value, if urser not provided
```

Comment:
typo for `user`

Reply:
instead of setting to `False` here, can you make it as a default value if not provided?

Reply:
I think not. Because I need give two values into `nic_sriov = vim.host.SriovConfig()`.
PyVmomi (VMware) requires two parameters. So, at this point it's better to set correct parameter.

Reply:
okay, seems right.
Just the typo correction for `user` is required.
Everything else looks good :)

Reply:
I did.
Anusha, thank you very much for your advices. Now it looks better )